util/text.py

```diff
@@ -52,7 +52,10 @@ def text_to_char_array(original, alphabet):
     Given a Python string ``original``, remove unsupported characters, map characters
     to integers and return a numpy array representing the processed string.
     """
-    return np.asarray([alphabet.label_from_string(c) for c in original])
+    integers = np.asarray([alphabet.label_from_string(c) for c in original])
+    if integers.shape[0] == 0:
+        raise Exception("Found an empty transcript! You must include a transcript for all training data.")
```

Comment:
It'd be better if this raised a more specific exception like `ValueError`. And even better if this exception was caught in the caller in `feeding.py` so it could print the corresponding `wav_filename` for the empty transcript as well. Then it'd be easier to find the problematic entry.